README.md

```diff
@@ -1,131 +1,126 @@
 ## Introduction
 
-`titus-isolate` applies isolation primitives to workloads running on a given agent in an attempt to control the impact of "noisy neighbors" depending on user requirements.
+`titus-isolate` enables the isolated and efficient use of compute resources in a multi-tenant container environment.
+Given a machine hosting multiple containers, `titus-isolate` partitions the available compute hardware (hyper-threads) 
+to achieve isolation and efficiency goals.
 
-## Install Dependencies
+In general greater efficiency implies less isolation and vice-versa.  To achieve maximal isolation workloads 
```

Comment:
this is unclear: are you talking about `cost` or `performance` efficiency?

Reply:
cost efficiency.  I'll update it.